### ISSUE TITLE
[SampleCard-ActivityUpdate; Config-testVariantHostConfig.json]: The color contrast ratio for the text is 2.9:1 which does not meet requirement of at least 4.5:1

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7394]
https://microsoft.visualstudio.com/OS/_workitems/edit/38020095
